editor/src/document/document_message_handler.rs

```diff
@@ -267,19 +276,19 @@ impl MessageHandler<DocumentsMessage, &InputPreprocessor> for DocumentsMessageHa
 				let prev_id = self.document_ids[prev_index];
 				responses.push_back(SelectDocument(prev_id).into());
 			}
-			Copy => {
+			Copy(clipboard) => {
 				let paths = self.active_document().selected_layers_sorted();
 				self.copy_buffer.clear();
 				for path in paths {
 					match self.active_document().graphene_document.layer(&path).map(|t| t.clone()) {
 						Ok(layer) => {
-							self.copy_buffer.push(layer);
+							self.copy_buffer[clipboard as usize].push(layer);
```

Comment:
How is the `clipboard` enum able to be cast to a `usize`?

Reply:
The enum is represented as an integer of the appropriate size which is then cast to a usize